### PR TITLE
Add a feeder and feeder list

### LINKED ISSUE #26: Feeder list
I want to have a view 
- accessible from the Drawer (side bar)
- Has a list of **Feeders**
- Shows an image of the feeder type (stock photo)
- Have a `+` button to add a Feeder

### PR DESCRIPTION
**Pull request by GuassianFlux**
Closes #26
Contributes to #23

### CODE REVIEW

**Review by jalobrano — DISMISSED** [review]

File: lib/ui/addFeeder/AddFeederScreen.dart [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        elevation: 0.0,
+        backgroundColor: Color(COLOR_PRIMARY),
+        title: Text('Update Feeder'),
+        leading: IconButton(
+          icon: new Icon(Icons.arrow_back_ios, color: Colors.white),
+          onPressed: () async {
+            Feeder temp = await FireStoreUtils().getFeeder(feeder.id);
```
Comment (wsciaroni) [inline lib/ui/addFeeder/AddFeederScreen.dart]: After creating a feeder and entering a serial number, I received a null exception on this line. Not sure what exactly happened. Maybe we need to verify non-null before referencing `temp`?

**Review by wsciaroni — COMMENTED** [review]
I noticed that after I created a feeder, there are two feeders in my feeder list. Is this an artifact from debugging?

I added a feeder named `kitchen` with some serial number. Then, I went to the feeder list and saw an additional feeder named `feeder` with an empty serial number.

**Review by jalobrano — APPROVED** [review]

**Comment by wsciaroni** [comment]
@GuassianFlux Did you verify this was fixed? I'm not at my machine to re-test, but I did re-review the code.

**Comment by jlp0063** [comment]
I have reviewed the code. As soon as we get confirmation that issues are fixed, I will be glad to approve. I see nothing wrong with the code, but just want confirmation of fixes that were asked for being made.

**Comment by GuassianFlux** [comment]
@wsciaroni 
I believe the issue was fixed. I'm unsure of how you first encountered the error. I was only able to reproduce it by closing the application during the editing process, and that issue has been addressed.

**Review by jlp0063 — APPROVED** [review]

**Comment by wsciaroni** [comment]
This issue appears to be resolved.

**Comment by wsciaroni** [comment]
I would like to see the ability to delete a feeder from ones account in the future.  I'll make an issue pointing to it

**Review by wsciaroni — APPROVED** [review]

**Pull request merged**